### ISSUE TITLE
Print fluxes

### ISSUE THREAD

**Issue by brunorbqi** [SysBioChalmers/RAVEN#318]
### Description of the issue:
I am following the tutorial 5 for the reconstruction of a metabolic model and having problem when I need to printFluxes
#### Reproducing this issue:
>> [fluxes, metabolite]=makeSomething(model,{'H+'},true);
model.metNames(metabolite)

ans =

  0×0 empty cell array

>> printFluxes(model, fluxes, false, [], [],'%rxnID (%rxnName):\n\t%eqn: %flux\n')
Error using dispEM (line 50)
The number of fluxes and the number of reactions must be the same

Error in printFluxes (line 69)
    dispEM(EM);
I
#### System information
* Please report:
1. RAVEN version 2.4
2. Operating system : Linux Ubuntu 20.04
**I hereby confirm that I have:**
- [x] Followed the [guidelines](https://github.com/SysBioChalmers/RAVEN/wiki/Installation) to install RAVEN.
- [x] Checked that a similar issue does not [already exist](https://github.com/SysBioChalmers/RAVEN/issues?utf8=%E2%9C%93&q=is%3Aissue)
- [x] If suitable, needed, asked first in the [Gitter chat room](https://gitter.im/SysBioChalmers/RAVEN) about the issue

**Comment by simas232** [comment]
@brunorbqi, was everything fine when you ran `checkInstallation`? There may be a problem with the solver software.

**Comment by brunorbqi** [comment]
Yes it was. I tried the tutorial 5 for Saccharomyces  and the print fluxes was ok. I am using the cobra solver by the way.

**Comment by edkerk** [comment]
@brunorbqi Do you mean that you get the error message when you run tutorial 5 for _another_ organism, and not for _Saccharomyces cerevisiae_ as specified in the tutorial? An issue could be that `H+` is not included in your model, although it seems unlikely that this would be the case with a KEGG-generated model, but might be worthwhile to check. What output do you get when you run `disp(model)` and `model.metNames(ismember(model.metNames,'H+'))` ?
